stable/mongodb/README.md

```diff
@@ -206,3 +211,14 @@ Use the workaround below to upgrade from versions previous to 5.0.0. The followi
 ```consoloe
 $ kubectl delete statefulset my-release-mongodb-arbiter my-release-mongodb-primary my-release-mongodb-secondary --cascade=false
 ```
+
+## Configure Ingress
+Currently we can expose mongodb externally using nginx-ingres. If you are using `nginx-ingress=1.3.x` check [this options](https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L383)
+on how to expose TCP ports. If you installed mongodb in `default` namespace, then an example would be:
+```yaml
+tcp:
+  27017: "default/mongodb:27017"
+```
+
+After that you will be able to access mongodb on the ingress endpoint without 
+specifying port.
```

Comment:
Without specifying the port? Do you mean using the port `80`?

Reply:
Yes, port 80.